src/time_integration/methods_SSP.jl

```diff
@@ -194,6 +194,10 @@ function solve!(integrator::SimpleIntegratorSSP)
 
         @. integrator.r0 = integrator.u
         for stage in eachindex(alg.c)
+            for stage_callback in alg.stage_callbacks
+                prepare_callback!(stage_callback, integrator)
+            end
```

Comment:
Will this work with the existing stage limiters?

Reply:
Hm I guess it makes more sense to remove this and simply use the integrators from OrdinaryDiffEq.jl